### PR TITLE
Ran erblint with autocorrect

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

It's good to use valid HTML

## Was the documentation (README, DevOpsDocs, wiki, consul, etc.) updated?

n/a

## Does this change affect how this application integrates with other services?
no
If so, please confirm change was tested on stage and/or test added to sul-dlss/infrastructure-integration-test.

### CODE REVIEW

**Comment by jcoyne** [comment]
@mjgiarlo I'm not sure how to make that happen.

**Comment by mjgiarlo** [comment]
@jcoyne :speech_balloon: 
> @mjgiarlo I'm not sure how to make that happen.

How did you run the linter?

**Comment by jcoyne** [comment]
`bundle exec erblint --lint-all --autocorrect`  but there are thousands of lint errors this does not address.

**Comment by jmartin-sul** [comment]
fine to merge, sort of assuming @mjgiarlo's request for changes has been resolved based on what i saw in the PR, but refraining from merging myself just in case i'm wrong.

File: app/views/content_types/_content_type.html.erb [hunk 1]
```diff
@@ -2,14 +2,22 @@
...
+  <% old_content_type = begin begin
+                                @object.contentMetadata.contentType[0]
+                              rescue StandardError
+                                ''
+                              end end %>
+  <% old_resource_type = begin begin
+                                 @object.contentMetadata.ng_xml.xpath('/contentMetadata/resource')[0]['type']
+                               rescue StandardError
+                                 ''
+                               end end %>
```
[the remark this answers is not in the document]
Comment (mjgiarlo) [reply app/views/content_types/_content_type.html.erb:15]: @jmartin-sul that looks busted to me. I'll push a change. Thanks!

[the remark this answers is not in the document]
Comment (mjgiarlo) [reply app/views/content_types/_content_type.html.erb]: Fixed, squashed, rebased on latest in `master`, and pushed. Resolving comment.

**Pull request merged**